src/AppBundle/Controller/BackupController.php

```diff
@@ -0,0 +1,204 @@
+
+        if (!$backup) {
+            throw new ElementNotFoundException(
+                'No Backup for id '.$id .' found'
+            );
+        }
+
+        $serializer = $this->get('jms_serializer');
+        $response = $serializer->serialize($backup, 'json');
+        return new Response($response);
+    }
+
+    /**
+     * Webhook to create a new Backup object
+     *
+     * @Route("/backups", name="create_backup_webhook", methods={"POST"})
+     *
+     * @param Request $request
+     * @return Response
+     * @throws ForbiddenException
+     *
+     * @OAS\Post(path="/backups?path={path}&token={token}",
+     * tags={"backups"},
+     * @OAS\Parameter(
+     *      description="The path to the backup file created by duplicity",
+     *      name="path",
+     *      in="query",
+     *      required=true,
+     *      @OAS\Schema(
+     *          @OAS\Property(
+     *              property="path",
+     *              type="string",
+     *          ),
+     *      ),
+     * ),
+     * @OAS\Parameter(
+     *      description="The authorization token set in the Backup Schedule",
+     *      name="token",
+     *      in="query",
+     *      required=true,
+     *      @OAS\Schema(
+     *          @OAS\Property(
+     *              property="token",
+     *              type="string",
+     *          ),
+     *      ),
+     * ),
+     * @OAS\Response(
+     *  description="The provided parameters are invalid or the path is missing",
+     *  response=400
+     * ),
+     * @OAS\Response(
+     *  description="The backup token is invalid",
+     *  response=403
+     * ),
+     * @OAS\Response(
+     *  description="The new Backup object was created",
+     *  response=201,
+     *  @OAS\JsonContent(ref="#/components/schemas/backup"),
+     * ),
+     * )
+     * @throws WrongInputException
+     * @throws WrongInputExceptionArray
+     */
+    public function backupCreationWebhook(Request $request){
+        $token = $request->query->get('token');
+
+        $backupSchedules = $this->getDoctrine()->getRepository(BackupSchedule::class)->findBy(["token" => $token]);
+
+        if (!$backupSchedules) {
+            throw new ForbiddenException(
+                'Invalid backup token'
+            );
+        }
+
+        //Token is unique, there can be only one result
+        $backupSchedule = $backupSchedules[0];
+
+        //Validate path
+        if (!$request->query->has('path')){
+            throw new WrongInputException(
+                'Backup file path is missing'
+            );
+        }
+
+        $path = $request->query->get('path');
+
+        $backup = new Backup();
+        $backup->setBackupSchedule($backupSchedule);
+        $backup->setFilePath($path);
+        $backup->setTimestamp();
+
+        if ($errorArray = $this->validation($backup)) {
+            throw new WrongInputExceptionArray($errorArray);
+        }
+
+        $em = $this->getDoctrine()->getManager();
```

Comment:
Wollten wir das nicht per injection über den Methoden Kopf in Hinblick auf 4.0 machen? Also dann einfach 
 `EntityManagerInterface $em` in den Methodenkopf einfügen.